src/cf-core/src/cf-base.less

```diff
@@ -291,6 +291,14 @@ h4,
     .h6 + & {
         margin-top: unit( 30px / @font-size, em );
     }
+
+    .respond-to-max( @bp-xs-max, {
+        @h4-font-size-on-xs: 16px;
```

Comment:
Presumably this should always match the body size, can we use `@base-font-size-px` here? Otherwise it's not very discoverable.

Reply:
Eh, I'm not 100% sure that's true, but it definitely should never drop _below_ the body size, so setting it to that would prevent it from doing so if, say, we ever increased the body size to 18. So yeah, I'll make that change.

Reply:
I was going off the helper text "displays at same size as body copy"

Reply:
Ah, I see. In that case, I was following the form of the bigger headings' descriptions, which say "displays at same size as h<-1>". The scenario I'm pondering where it might not be true in the future is if we decided we wanted to adjust the hierarchy so that all four levels and the ghost-level for h4s on mobile were above the body size. But this will do for now.

Reply:
Updates pushed. We could conceivably do away with the `@h4-font-size-on-xs` variable at this point and just refer to `@base-font-size-px` in the three properties, or we could leave it in place in case the scenario I described ever comes to pass. Do you have a strong preference?